RELEASE_NOTES.rst

```diff
@@ -1,3 +1,11 @@
+coala-bears 0.10.2
```

Comment:
I assume since we are backporting we are making a release, hence added the release notes.